reporter/build.gradle.kts

```diff
@@ -80,6 +81,7 @@ dependencies {
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion")
     implementation("com.squareup.retrofit2:retrofit:$retrofitVersion")
     implementation("com.vladsch.flexmark:flexmark:$flexmarkVersion")
+    implementation("net.sf.saxon:Saxon-HE:$saxonHeVersion")
```

Comment:
I believe this should *replace* line 96 below, so that `Saxon-HE` is used in *instead* of `xalan`. The comment down there can probably stay as-is, but it might be that the expected output of `StaticHtmlReporterFunTest` needs to be adjusted.

@rbieniek, do you want to work on this, or do you want me to take over, making the additional changes?

Reply:
Also see https://bz.apache.org/bugzilla/show_bug.cgi?id=65700, BTW.

Reply:
Sorry, I got impatient and addressed the comments myself, see https://github.com/oss-review-toolkit/ort/pull/5033.